jorge/css/style.css

```diff
@@ -0,0 +1,30 @@
+.nav_container {
```

Comment:
BEM convention uses double underscore, you have single underscore in all your classes